book/src/configuration.md

```diff
@@ -180,6 +180,8 @@ auto-pairs = false # defaults to `true`
 The default pairs are <code>(){}[]''""``</code>, but these can be customized by
 setting `auto-pairs` to a TOML table:
 
+Example
+
```

Comment:
Did you intend to add this?

Reply:
Sorry, that's accidental.

Reply:
Resolved by https://github.com/helix-editor/helix/pull/5768/commits/4fe98965f3389a8f7757b94061aa41f5e0a2c5ab